fxa-client/sdks/swift/FxAClient/FxAClient.xcodeproj/project.pbxproj

```diff
@@ -329,6 +330,7 @@
 				SDKROOT = iphoneos;
 				SWIFT_COMPILATION_MODE = wholemodule;
 				SWIFT_OPTIMIZATION_LEVEL = "-O";
+				SWIFT_VERSION = 4.2;
```

Comment:
not sure if we needed this, but otherwise it fallsback to default xcode version or something

Reply:
👍